### PR TITLE
[next] (Doc) Update the profile config doc

### PR DESCRIPTION
**Pull request by IgorCATech**
Signed-off-by: Igor Kazmyr <igor.kazmyr@broadcom.com>

## Proposed changes

Addressing the #1392 issue. Adding information about the global and project-level profile configuration

## Release Notes

Milestone:

Changelog:

## Types of changes

What types of changes does your code introduce to Zowe Explorer?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Updates to Documentation or Tests (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This checklist will be used as reference for both the contributor and the reviewer_

- [x] I have read the [CONTRIBUTOR GUIDANCE](https://github.com/zowe/vscode-extension-for-zowe/wiki/Best-Practices:-Contributor-Guidance) wiki
- [ ] PR title follows [Conventional Commits Guidelines](https://www.conventionalcommits.org/en/v1.0.0-beta.2/)
- [x] PR Description is included
- [ ] gif or screenshot is included if visual changes are made
- [ ] `yarn workspace vscode-extension-for-zowe vscode:prepublish` has been executed
- [ ] All checks have passed (DCO, Jenkins and Code Coverage)
- [ ] I have added unit test and it is passing
- [ ] I have added integration test and it is passing
- [ ] There is coverage for the code that I have added
- [ ] I have tested it manually and there are no regressions found
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any PR dependencies have been merged and published (if appropriate)

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by IgorCATech** [comment]
Any feedback/updates here?

**Comment by crawr** [comment]
Looks good, @IgorCATech ! I just added a commit to provide a sample configuration file. Kindly check as well. Thanks!

File: docs/Early Access - Using Global Profile Configuration.md [hunk 1]
```diff
@@ -1,36 +1,131 @@
...
+3. Reload your VS Code window by clicking **View** > **Command Palette** > **Developer: Reload Window** so that the changes take effect.
+
+You successfully edited your configuration profile.
+
+### Sample Profile Configuration
+
+View the profile configuration sample. In the sample, the default `lpar1.zosmf` profile will be loaded upon activation.
 
-### Add, Update, and Delete a Profile
+You can use the sample to customize you profile configuration file. Ensure that you edit the `host` and `port` values before you work in your environment.
```
Comment (crawr) [inline docs/Early Access - Using Global Profile Configuration.md:71]: small typo you -> your profile configuration file

Comment (IgorCATech) [reply docs/Early Access - Using Global Profile Configuration.md:71]: oh... Nice catch! let me fix that

**Review by crawr — DISMISSED** [review]

**Comment by jellypuno** [comment]
@crawr @IgorCATech Do we need to lock-in a specific version of the @next? or just mentioning @next is enough?

File: docs/Early Access - Using Global Profile Configuration.md [hunk 2]
```diff
@@ -1,36 +1,131 @@
...
+You can configure global profiles and project-specific profiles.
 
 ## Prerequisites
 
-- Install [Zowe CLI @next version](https://github.com/zowe/zowe-cli/blob/next/docs/Early%20Access%20-%20Using%20Global%20Profile%20Configuration.md#installing-next-version).
-- [Initialize](https://github.com/zowe/zowe-cli/blob/next/docs/Early%20Access%20-%20Using%20Global%20Profile%20Configuration.md#initializing-global-configuration) the Global Configuration file. The resulting `zowe.config.json` and `zowe.schema.json` files will be placed in your Zowe home directory.
-- [Customize](https://github.com/zowe/zowe-cli/blob/next/docs/Early%20Access%20-%20Using%20Global%20Profile%20Configuration.md#editing-configuration) the Global Configuration file.
+Meet the following software requirements before you use the team configuration file:
+
+1. Install [Zowe CLI @next version](https://github.com/zowe/zowe-cli/blob/next/docs/Early%20Access%20-%20Using%20Global%20Profile%20Configuration.md#installing-next-version).
```
Comment (jellypuno) [inline docs/Early Access - Using Global Profile Configuration.md:13]: @crawr Can we add the list of supported CLI version here? Maybe a side by side list of the ZE version and its CLI version counterpart

Comment (crawr) [reply docs/Early Access - Using Global Profile Configuration.md:13]: | Zowe Explorer Version    | Zowe SDK Version        |
|--------------------------|-------------------------|
| v2.0.0-next.202107151328 | 7.0.0-next.202106012053 |
| TBD                      | 7.0.0-next.202108202027 |

**Review by crawr — APPROVED** [review]

**Review by jellypuno — APPROVED** [review]

**Pull request merged**